prefect_saturn/core.py

```diff
@@ -149,6 +147,7 @@ def _set_flow_metadata(self, flow: Flow) -> None:
         self._saturn_flow_id = str(response_json["id"])
         self._saturn_flow_version_id = response_json["flow_version_id"]
         self._saturn_image = response_json["image"]
+        self._saturn_flow_labels = response_json.get("labels", ["saturn-cloud"])
```

Comment:
Set this with a default to allow us to release prefect-saturn without worrying about customers that haven't yet been updated.

Reply:
oooooo good idea! Seems so obvious now that I see it haha

Reply:
now it can just be a patch version (`0.4.1`)